### PR TITLE
Mm 16623 fix all initialism errors in dir channels/store/storetest/

### PR DESCRIPTION
**Pull request by TealWater**
#### Summary

Fixed all initialism errors in 13 files in accordance with the golang style doc found [here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

List of warnings fixed:

variable naming conventions --> abbreviations like ID need to be capitalized
```
var-naming: var user_ID should be useID (revive)
var-naming: var teanId should be teamID (revive)

```

#### Ticket Link

Fix for: https://github.com/mattermost/mattermost/issues/16623

#### Release Note

```release-note
none

```

### CODE REVIEW

File: server/channels/store/storetest/channel_store.go [hunk 1]
```diff
@@ -371,13 +371,19 @@ func testChannelStoreUpdate(t *testing.T, rctx request.CTX, ss store.Store) {
...
 	uid := model.NewId()
+<<<<<<< HEAD
 	m1 := &model.TeamMember{TeamId: teamId1, UserId: uid}
 	m2 := &model.TeamMember{TeamId: teamId2, UserId: uid}
 	_, nErr := ss.Team().SaveMember(rctx, m1, -1)
+=======
+	m1 := &model.TeamMember{TeamId: teamID1, UserId: uid}
+	m2 := &model.TeamMember{TeamId: teamID2, UserId: uid}
+	_, nErr := ss.Team().SaveMember(m1, -1)
+>>>>>>> fixed var-naming issues identified by go.lint-revive in 13 files
```
Comment (hanzei) [inline server/channels/store/storetest/channel_store.go:386]: There is a merge conflict to resolve here.

**Review by hanzei — CHANGES_REQUESTED** [review]
Let's first fix CI before jumping in a propper review

File: server/channels/store/storetest/channel_store.go (hunk 1 above)
Comment (TealWater) [reply server/channels/store/storetest/channel_store.go:386]: My bad, fixed.

**Review by hanzei — CHANGES_REQUESTED** [review]
There are still some merge errors

`Error: channels/store/storetest/channel_store.go:378:1: syntax error: unexpected <<, expected }`
See https://github.com/mattermost/mattermost/actions/runs/8328279025/job/22821573263?pr=26461

**Review by hanzei — CHANGES_REQUESTED** [review]
We are getting there :smile:  https://github.com/mattermost/mattermost/actions/runs/8343895289/job/22840651836?pr=26461

**Comment by TealWater** [comment]
Meh, I'll kill this PR and redo it, this error is beyond me.

I'll just re-fork the repo and start fresh/

**Comment by hanzei** [comment]
That is a bummer. @TealWater Please let me know if there is anything I can help you with.

**Pull request closed without merging**